xbmc/filesystem/BlurayDirectory.h

```diff
@@ -42,8 +47,15 @@ class CBlurayDirectory : public IDirectory
 
   void         Dispose();
   std::string  GetDiscInfoString(DiscInfo info);
-  void         GetRoot  (CFileItemList &items);
+  void GetRoot(CFileItemList& items);
+  void GetRoot(CFileItemList& items,
+               const CFileItem& episode,
+               const std::vector<CVideoInfoTag>& episodesOnDisc);
+  void AddRootOptions(CFileItemList& items);
   void         GetTitles(bool main, CFileItemList &items);
+  void GetEpisodeTitles(const CFileItem& episode,
+                        CFileItemList& items,
+                        const std::vector<CVideoInfoTag>& episodesOnDisc);
```

Comment:
Please check whether some of these new methods could be const.

Reply:
AddRootOptions seems to be the only one. Done.